packages/web-pixels-extension/src/types/PixelEvents/index.ts

```diff
@@ -581,6 +581,7 @@ export interface Collection {
    * A globally unique identifier.
    */
   id: string;
+  productVariants: ProductVariant[];
```

Comment:
If you run `yarn:schem all` does this get removed? 

I don't believe you change it here. This file should be generated.

Reply:
this was generated by my edits to Collection running `yarn:schema all`